src/styles.css

```diff
@@ -7,18 +7,67 @@ main {
   overflow: hidden;
 }
 
+header {
+  width: 100%;
+  height: 75px;
+}
+
 .flex {
   display: flex;
 }
 
 .row {
   flex-direction: row;
+  align-items: center;
+}
+
+.end {
+  flex-direction: row-reverse;
 }
 
 .column {
   flex-direction: column;
 }
 
+.around {
+  justify-content: space-around;
+}
+
+header > section {
```

Comment:
I like this method over giving everything an individual class, excited to see how it pans out in the large scheme